src/utils/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const SHORT_LINKS_ROUTE_PREFIX = '/short-links';
```

Comment:
Should this file be in `services/shortLink`?

Reply:
Yes, it could be. I'll move their in shortLink service.